www/nginx/src/opnsense/mvc/app/controllers/OPNsense/Nginx/forms/location.xml

```diff
@@ -61,6 +61,12 @@
     <style>selectpicker</style>
     <help>Select an upstream to proxy to or connect via FastCGI if chosen.</help>
   </field>
+  <field>
+    <id>location.path_prefix</id>
+    <label>Path prefix</label>
+    <type>text</type>
+    <help>Define an optional path prefix for this location.</help>
```

Comment:
Please add 'It will be prepended to the path so http://www.example.com/index.php will be sent to http://www.example.com/app/index.php on your upstream if you choose "/app" here'.